packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -283,7 +283,7 @@ export const InlineEditingMixin = (superClass) =>
         )[0];
 
         if (overlay) {
-          overlay.addEventListener('vaadin-overlay-outside-click', this.__boundEditorFocusOut);
+          overlay.addEventListener('vaadin-overlay-closing', this.__boundEditorFocusOut);
```

Comment:
This change seems not to be covered by unit tests. I checked it locally: the grid-pro unit tests pass when I change the event back to `vaadin-overlay-outside-click`.

Reply:
Good catch. Seems that [this change](https://github.com/vaadin/web-components/pull/3138/commits/a46c235a2f72461f31e67990b6079d1f0a6930b4#diff-a22b589817f0a0a072a03603aacbf01899a28200af52529f4d22879f213a5776L691-R693) in overlay made it unnecessary to change the event type here. Reverted.